External/FEXCore/Source/Interface/Core/Interpreter/MemoryOps.cpp

```diff
@@ -288,6 +288,111 @@ DEF_OP(StoreMem) {
   }
 }
 
+DEF_OP(MemSet) {
+  const auto Op = IROp->C<IR::IROp_MemSet>();
+  const int32_t Size = Op->Size;
+
+  char *MemData = *GetSrc<char **>(Data->SSAData, Op->Addr);
+  const auto Value = *GetSrc<uint64_t*>(Data->SSAData, Op->Value);
+  const auto Length = *GetSrc<uint64_t*>(Data->SSAData, Op->Length);
+  const auto Direction = *GetSrc<uint8_t*>(Data->SSAData, Op->Direction);
+
+  auto MemSetElements = []<typename T>(T* Memory, uint64_t Value, size_t Length) {
```

Comment:
No need for explicit `typename T` in these lambdas. `auto* Memory` also works to ensure the argument is a pointer.

Reply:
Nice. Removed lambda template.